util/dvsim/dvsim.py

```diff
@@ -402,6 +402,12 @@ def parse_args():
                         help=('The options for each build_mode in this list '
                               'are applied to all build and run targets.'))
 
+    disg.add_argument("--gui",
+                      action='store_true',
+                      default=False,
```

Comment:
Nit: `store_true` already sets the default value, so you don't actually need that here.

Reply:
Updated.